paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -116,11 +158,39 @@ def create_instance_uwsgi_scaling_rule(
     """
     Creates a Prometheus adapter rule config for a given service instance.
     """
+    setpoint = instance_config["autoscaling"].get(
+        "setpoint", DEFAULT_AUTOSCALING_SETPOINT
+    )
+    moving_average_window = instance_config["autoscaling"].get(
+        "moving_average_window_seconds", DEFAULT_AUTOSCALING_MOVING_AVERAGE_WINDOW
+    )
```

Comment:
if you loaded the config into a KubernetesDeploymentConfig and added `moving_average_window_seconds` to the default in https://github.com/Yelp/paasta/pull/3023/files#diff-719160feb410077ed31b8c62c719bf6d2495cbd251119458fb74ad4aea59942fR335, you could DRY this default.

Reply:
done in https://github.com/Yelp/paasta/pull/3023